### ISSUE TITLE
Need interface for `SparsePSHDegreeM`

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACE1.jl#61]
This looks horrible and cannot be exposed to end-users. Need to fix this at a minimum for the simplified interfaces.
```julia
    wY = 1.5
    maxdeg = Dict(1 => 25, 2 => 23, 3 => 20, 4 => 10)
    D = SparsePSHDegreeM(Dn = Dict("default" => 1.0), 
                         Dl = Dict("default" => wY), 
                         Dd = maxdeg)
    acebasis = acemodel(species = [:Ti, :Al], N = 4, 
                        D = D, 
                        maxdeg = 1)

```

**Comment by cortner** [comment]
see PR #63

**Comment by cortner** [comment]
closed by #63
